the_shards_command/README.md

```diff
@@ -0,0 +1,128 @@
+# The Shards command
+
+Crystal is typically accompanied by its dependency manager `shards`.
+
+It manages dependencies for Crystal projects and libraries with reproducible
+installs across computers and systems.
+
+## Installation
+
+Shards is usually distributed with Crystal itself. Alternatively, a separate `shards` package may be available for your system.
+
+To install from source, download or clone [the repository](https://github.com/crystal-lang/shards) and run `make CRFLAGS=--release`. The compiled binary is in `bin/shards` and should be added to `PATH`.
+
+## Usage
+
+`shards` requires the presence of a `shard.yml` file in the project folder (working directory). This file describes the project and lists dependencies that are required to build it.
+A default file can be created by running [`shards init`](#shards-install).
+The file's contents are explained in the [*Writing a Shard* guide](../guides/writing_shards.md) and a detailed description of the file format is provided by the [shard.yml specification](https://github.com/crystal-lang/shards/blob/master/SPEC.md).
+
+Running [`shards install`](#shards-install) resolves and installs the specified dependencies.
+The installed versions are written into a `shard.lock` file for using the exact same dependency versions when running `shards install` again.
+
+If your shard builds an application, both `shard.yml` and `shard.lock` should be checked into version control to provide reproducible dependency installs.
+If it is only a library for other shards to depend on, `shard.lock` should *not* be checked in, only `shard.yml`. It's good advice to add it to `.gitignore` (the [`crystal init`](../using_the_compiler/README.md#crystal-init) does this automatically when initializing a `lib` repository).
+
+## Shards commands
+
+```bash
+shards [<options>...] [<command>]
```

Comment:
I'd like to keep a uniform structure to all the commands. These examples only look redundant because these commands don't accept any special arguments.